components/eam/src/chemistry/modal_aero/modal_aero_drydep.F90

```diff
@@ -733,4 +615,380 @@ subroutine calcram(ncol,landfrac,icefrac,ocnfrac, &
+                stickfrac = max( stickfrac_lowerbnd, exp(-sqrt(stk_nbr)) )
+             endif
+
+             !----------------------------------------------------------------------------------
+             ! Using the numbers calculated above, compute the quasi-laminar layer resistance
+             ! following Zhang L. et al. (2001), Eq. (5)
+             !----------------------------------------------------------------------------------
+             rss_lmn = 1.0_r8 / (eps0 * fricvel(ii) * stickfrac * (brownian+interception+impaction))
+
+             !--------------------------------------------------------------------
+             ! Total resistence and deposition velocity of turbulent deposition,
+             ! see Eq. (21) of Zender et al. (2003)
+             !--------------------------------------------------------------------
+             rss_trb = ram1(ii) + rss_lmn + ram1(ii)*rss_lmn*vlc_grv(ii)
+             vlc_trb_ontype = 1.0_r8 / rss_trb
+
+             !--------------------------------------------------------------------------------
+             ! Contributions to the single-value bulk deposition velocities of the grid cell
+             !--------------------------------------------------------------------------------
+             vlc_trb_wgtsum = vlc_trb_wgtsum + lnd_frc*( vlc_trb_ontype )
+             vlc_dry_wgtsum = vlc_dry_wgtsum + lnd_frc*( vlc_trb_ontype + vlc_grv(ii) )
+          endif
+
+       enddo  ! lt=1,n_land_type
+
+       vlc_trb(ii) = vlc_trb_wgtsum
+       vlc_dry(ii) = vlc_dry_wgtsum
+
+    enddo ! ii=1,ncol
+    !-------
+
+  end subroutine modal_aero_turb_drydep_velocity
+
+  !==============================================================================
+  ! Calculate the radius for a moment of a lognormal size distribution
+  !==============================================================================
+  real(r8) function radius_for_moment( moment,sig_part,radius_part,radius_max)
+
+    integer, intent(in) :: moment        ! moment of the particle size distribution. 0 = number; 3 = volume
+    real(r8),intent(in) :: sig_part      ! geometric standard deviation of particle size distribution
+    real(r8),intent(in) :: radius_part   ! mean (volume or number) particle radius [m]
+    real(r8),intent(in) :: radius_max    ! developer-specified upper bound of mean radius [m]
+
+    real(r8) :: lnsig
+    
+    lnsig = log(sig_part)
+    radius_for_moment = min(radius_max,radius_part)*exp((float(moment)-1.5_r8)*lnsig*lnsig)
+
+  end function radius_for_moment
+
+  !==========================================================================
+  ! Calculate dynamic viscosity of air, unit [kg m-1 s-1]. See RoY94 p. 102
+  !==========================================================================
+  real(r8) function air_dynamic_viscosity( temp )
+
+    real(r8),intent(in) :: temp   ! air temperature [K]
+
+    air_dynamic_viscosity = 1.72e-5_r8 * ( (temp/273.0_r8)**1.5_r8) * 393.0_r8 &
```

Comment:
I just saw a different calculation of air dynamic viscosity in wet deposition (see variable `airdynvisc` in `aero_model.F90`). I will create a similar function and make a note of the difference.

Reply:
I just calculated both formula. It seems that there is a 10x difference. It might be a potential bug.

Here the result is 1.4986e-6 * (temp)**1.5 / (temp+120.0) while in aero_model.F90 it is
1.4963e-5 * (temp)**1.5 / (temp_120.0)

Reply:
thank you @mingxuanwupnnl for checking this. Just don't know why the equations look different and I don't find a reference in wet deposition. for example: this one uses `(temp/296.16_r8)**1.5_r8` but in dry deposition is `(temp/273.0_r8)**1.5_r8)`? The 10x difference may due to the unit different as in wet deposition it is `g/cm/s` while in dry deposition it is `kg/m/s`.

Reply:
Thank you, @tangshuaiqi The unit difference make senses. 296.16 vs 273 is probably for different reference temperature. That's my best guess. For dry dep, we can verify it from the Rogers and Yau's book which refers to.